### ISSUE TITLE
How to export roughness / metalness / aoMapIntensity

### ISSUE THREAD

**Issue by donalffons** [KhronosGroup/glTF-Blender-IO#646]
Hi,

I did a lot of research, but couldn't find a way to export roughness / metalness / aoMapIntensity factors (in contrast to maps). In the old gltf-Blender-Exporter, these factors were directly accessible through the gtlf node.

For example, here is my material setup:

![image](https://user-images.githubusercontent.com/33040347/64072512-fbe52980-cc8f-11e9-8c9a-85a451ee0e4e.png)

I would like to have more control over the roughness and ambient occlusion maps and also change the factor. Is there a way to do that? I tried inserting math multiply nodes and rgb multiply nodes at various locations without success. As a workaround I am currently using userData / custom properties in Blender.

I would greatly appreciate it if someone could point out if I am missing something. If this feature is currently missing, having it implemented would be a great addition.

Cheers
-Sebastian

**Comment by donmccurdy** [comment]
Adding a multiply node should be supported, [according to the docs](https://docs.blender.org/manual/en/latest/addons/io_scene_gltf2.html#factors). Could you share a screenshot of what you're trying for that? We should probably add a screenshot of the correct setup in the docs, but I think it's multiplication _after_ the *Separate RGB* node.

**Comment by donalffons** [comment]
I tried the "Math - Multiply" and the "MixRGB - Multiply" nodes, both before and after the "Separate RGB" node without success.
![image](https://user-images.githubusercontent.com/33040347/64091560-00bde200-cd51-11e9-93c2-2f8955c75ced.png)

When loading the GLTF file with ThreeJS, the roughness is always 1 no matter what I do.

I created a simple test file (see attached - I'm using some random AO texture as a roughness map, that's not by mistake). In the exported GLTF, my roughness factor is not included.

[gltf bugreport.zip](https://github.com/KhronosGroup/glTF-Blender-IO/files/3564552/gltf.bugreport.zip)
